components/Navigation/index.js

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import styled from "styled-components";
+import "material-symbols";
```

Comment:
```suggestion
import styled from "styled-components";
```

You could remove unused imports.

Reply:
Done.